core/src/main/scala/org/elasticmq/actor/queue/QueueMetadataListener.scala

```diff
@@ -0,0 +1,7 @@
+package org.elasticmq.actor.queue
+
+import org.elasticmq.QueueData
+
+case class PersistQueue(queue: QueueData)
```

Comment:
maybe group these inside an `object QueueMetadataListener`. We won't pollute global scope then. Unless other actors are designed otherwise :)

Reply:
From what I see for `QueueManagerActor` and `QueueActor` messages are not grouped in any way.

Reply:
Ok, maybe a trait which would be extended by all messages accepted by the actor? So that the protocol is clear